druid/src/widget/container.rs

```diff
@@ -135,21 +135,14 @@ impl<T: Data> Widget<T> for Container<T> {
     }
 
     fn paint(&mut self, ctx: &mut PaintCtx, data: &T, env: &Env) {
+        let panel = ctx.size().to_rounded_rect(self.corner_radius);
```

Comment:
This shouldn't be moved out of the `if let Some(background) = self.background.as_mut() {}` block.